app.js

```diff
@@ -15,9 +15,15 @@ var downloadVideo = function(params, callback) {
 		.on('finish', callback);
 
 	// test
-	/*setTimeout(function() {
-		callback();
-	}, Math.ceil(Math.random()*30));*/
+	/*
+	setTimeout(function() {
+	callback();
+	}, Math.ceil(Math.random()*30));
+	*/
+}
+
+var filterFileName = function(fileName){
+	return (fileName.replace(/[\\\/\:\*\?\"\<\>\|]+/, "").replace(/\s+/, " "));
```

Comment:
You regular expressions in `replace` will change only first occurrence. Use 'g' option to replace all occurrences.